test_scripts/API/SetAudioStreamingIndicator/033_ATF_N_TC_SetAudioStreamingIndicator_media_app_HMI_replies_resultcode_success_false.lua

```diff
@@ -0,0 +1,111 @@
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.application1.registerAppInterfaceParams.appHMIType = {"MEDIA"}
+config.application1.registerAppInterfaceParams.isMediaApplication = true
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+
+--[[ Local variables ]]
+-- in scope of the CRQ info parameter is not specified, but will be left for any future use.
+local hmi_result_code = {
+	{ result_code = "UNSUPPORTED_REQUEST", info = "" },
+	{ result_code = "DISALLOWED", info = "" },
+	{ result_code = "USER_DISALLOWED", info = "" },
+	{ result_code = "REJECTED", info = "" },
+	{ result_code = "ABORTED", info = "" },
+	{ result_code = "IGNORED", info = "" },
+	{ result_code = "IN_USE", info = "" },
+	{ result_code = "DATA_NOT_AVAILABLE", info = "" },
+	{ result_code = "TIMED_OUT", info = "" },
+	{ result_code = "INVALID_DATA", info = "" },
+	{ result_code = "CHAR_LIMIT_EXCEEDED", info = "" },
+	{ result_code = "INVALID_ID", info = "" },
+	{ result_code = "DUPLICATE_NAME", info = "" },
+	{ result_code = "APPLICATION_NOT_REGISTERED", info = "" },
+	{ result_code = "OUT_OF_MEMORY", info = "" },
+	{ result_code = "TOO_MANY_PENDING_REQUESTS", info = "" },
+	{ result_code = "GENERIC_ERROR", info = "" },
+	{ result_code = "TRUNCATED_DATA", info = "" }
+}
+
+--[[ General Precondition before ATF start ]]
+testCasesForPolicyTable:precondition_updatePolicy_AllowFunctionInHmiLeves({"BACKGROUND", "FULL", "LIMITED"}, "SetAudioStreamingIndicator")
+commonSteps:DeleteLogsFiles()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_ActivateApp()
+  commonSteps:ActivateAppInSpecificLevel(self, self.applications[config.application1.registerAppInterfaceParams.appName])
+  EXPECT_NOTIFICATION("OnHMIStatus", {systemContext = "MAIN", hmiLevel = "FULL"})
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+for i = 1, #hmi_result_code do
+	Test["TestStep_SetAudioStreamingIndicator_"..hmi_result_code[i].result_code.."_audioStreamingIndicator_PAUSE"] = function(self)
+	  local corr_id = self.mobileSession:SendRPC("SetAudioStreamingIndicator", { audioStreamingIndicator = "PAUSE" })
+
+	  EXPECT_HMICALL("UI.SetAudioStreamingIndicator", { audioStreamingIndicator = "PAUSE" })
+	  :Do(function(_,data) 
+	  	-- ATF issue: "Genivi: SDL doesn't accept some error_codes from HMI when they are sending with type of protocol_message "error""
```

Comment:
@dboltovskyi, yes it can be used:
self.hmiConnection:Send('{"id":'..tostring(data.id)..',"jsonrpc":"2.0","result":{"method":"UI.SetAudioStreamingIndicator", "audioStreamingIndicator":"PLAY", "code":'..tostring(value)..'}}'), where value is taken from HMI_API.xml
<element name="UNSUPPORTED_REQUEST" value="1"/>

But I don't prefer to use this in scope of the CRQ, because in this way we are using "result", which should be used for success: true, and we should have error message.